### PR TITLE
docs: add state-based migration docs

### PR DESCRIPTION
**Pull request by rebelice**
close BYT-2725

### CODE REVIEW

File: content/docs/change-database/state-based-migration/one-click-to-dump-sdl-schema.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: One-click to Dump SDL Schema
+---
+
+Dumping an SDL format of the database schema from Bytebase to GitOps configuration's LATEST file can save time and improve efficiency, as it eliminates the need to manually write the schema. You may use it when initializing the LATEST file or when the LATEST file needs to be updated due to being outdated.
+
+## How to Dump
```
Comment (tianzhou) [inline content/docs/change-database/state-based-migration/one-click-to-dump-sdl-schema.md:7]:
```suggestion
## How to Baseline
```

File: content/docs/change-database/state-based-migration/one-click-to-dump-sdl-schema.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: One-click to Dump SDL Schema
+---
+
+Dumping an SDL format of the database schema from Bytebase to GitOps configuration's LATEST file can save time and improve efficiency, as it eliminates the need to manually write the schema. You may use it when initializing the LATEST file or when the LATEST file needs to be updated due to being outdated.
```
Comment (tianzhou) [inline content/docs/change-database/state-based-migration/one-click-to-dump-sdl-schema.md:5]:
```suggestion
You will baseline the SDL schema in the following cases:

1. Initialize the LATEST file in the linked repository.
1. Update the LATEST file to reconcile the actual database schema.
```

**Comment by rebelice** [comment]
@tianzhou @Candybase PTAL

**Comment by adela-bytebase** [comment]
Agree with @Candybase 's proposal, however, we can't handle 4-level titles and what to put under Migration-based section.
Maybe we could explain that there are two migrations types on "State-based Migration" page? and also link this article  https://www.bytebase.com/blog/database-version-control-state-based-vs-migration-based

**Review by tianzhou — APPROVED** [review]
LGTM. We'd like to highlight SDL, so it's Ok to break the hierarchy and move it up.

**Comment by rebelice** [comment]
> Agree with @Candybase 's proposal, however, we can't handle 4-level titles and what to put under Migration-based section. Maybe we could explain that there are two migrations types on "State-based Migration" page? and also link this article https://www.bytebase.com/blog/database-version-control-state-based-vs-migration-based

@adela-bytebase I added migration type description at [enable-gitops-workflow.md](https://github.com/bytebase/bytebase.com/pull/845/files#diff-36c7779d89a5efa1b5c40d5d1ca1183a85b8323d574402674d840f3584b37855)

**Pull request merged**